src/components/Signup/SignupHeader.js

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react'
+import { Header, Text, NavBar, Icon, Button } from '../Reusable'
+import { getTextColor, getBgColor, getIconColor } from '../../utils'
+import { useTheme } from '../../contexts'
+import { Link } from 'react-router-dom'
+
+const SignupHeader = () => {
+    const { theme, toggleTheme } = useTheme()
+    const [isSmallNavVisible, setIsSmallNavVisible] = useState(false)
+
+    function toggleNavVisibility() {
+        setIsSmallNavVisible(!isSmallNavVisible)
+    }
+
+    return (
+        <Header>
+
+            <Link to='/'>
+                <Text classes='txt-lg txt-ucase'>
+                    sneakerviews
+                </Text>
+            </Link>
+
+            <NavBar id='nav-big' classes='flx-min-center'>
+
+                <Link to='/videos'>
+                    <Button classes={`btn-txt ${getTextColor(theme)} txt-md txt-lcase mg-right-lg`}>
+                        browse
+                    </Button>
+                </Link>
+
+                <Link to='/playlists'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        playlist_play
+                    </Icon>
+                </Link>
+
+                <Link to='/history'>
+                    <Icon classes={`${getIconColor(theme)} mg-right-lg`}>
+                        history
+                    </Icon>
+                </Link>
+
+                <Link to='/login'>
+                    <Button classes={`btn-txt txt-lcase ${getTextColor(theme)} bg-primary pd-xs txt-md`}>
+                        login
+                    </Button>
```

Comment:
(..cont) + this part.
both login + signup can reuse the same header component. You can conditionally render login/signup button based on the path...

Reply:
thanks for this feedback. I'll surely include it.